tests/test_json_model.py

```diff
@@ -290,6 +290,14 @@ def test_no_data_wrapper_deserialize(self):
         content = model.response(resp, content)
         self.assertEqual(content, {"data": "is good"})
 
+    def test_no_data_wrapper_deserialize_text_format(self):
+        model = JsonModel(data_wrapper=False)
+        resp = httplib2.Response({"status": "200"})
+        resp.reason = "OK"
+        content = CSV_TEXT_MOCK
+        content = model.response(resp, content)
+        self.assertEqual(content, CSV_TEXT_MOCK)
+
```

Comment:
We need test coverage for the "oddball" cases:
- Users passes a non-`str` as `content`.
- `content` is valid JSON, but doesn't have a `data` key
- `content` has the expected `data` key, but the model has `data_wrapper` set to `False`.

Reply:
- Ok for the first case.
- Doesn't the `test_data_wrapper_deserialize_nodata` test (the last one) address the second case?
- Ok for the third, but it feels very "oddball" :)

Reply:
Ah, the third case seems to be addressed with `test_no_data_wrapper_deserialize` test.